docs/admin/index.md

```diff
@@ -119,26 +119,27 @@ You can see an example of using some of the environment variables in this
 
 #### perseo-fe
 
-| Environment variable      | Default Value | Description                                                         |
-| :------------------------ | :------------ | :------------------------------------------------------------------ |
-| `PERSEO_ENDPOINT_HOST`    |               | Host where the CEP will listen to.                                  |
-| `PERSEO_ENDPOINT_PORT`    |               | Port where the CEP will listen to.                                  |
-| `PERSEO_MONGO_ENDPOINT`✅ |               | Endpoint (host[:port]) list for Mongo DB.                           |
-| `PERSEO_MONGO_REPLICASET` |               | ReplicaSet name for Mongo DB.                                       |
-| `PERSEO_MONGO_USER`       |               | User for Mongo DB.                                                  |
-| `PERSEO_MONGO_PASSWORD`   |               | Password for Mongo DB.                                              |
-| `PERSEO_CORE_URL`✅       |               | URL where Perseo Core is listening (e.g: http://host_or_ip:port).   |
-| `PERSEO_NEXT_URL`         |               | URL where Perseo Core replicated node is listening.                 |
-| `PERSEO_ORION_URL`✅      |               | URL of the Orion Context Broker.                                    |
-| `PERSEO_LOG_LEVEL`        |               | Log level.                                                          |
-| `PERSEO_SMTP_HOST`        |               | Host of the SMTP server                                             |
-| `PERSEO_SMTP_PORT`        |               | Port of the SMTP server                                             |
-| `PERSEO_SMTP_VERIFY_CA`   | `false`       | `true` if self-signed or invalid TLS certificate should be rejected |
-| `PERSEO_SMTP_SECURE`      | `false`       | `true` if SSL should be used with the SMTP server                   |
-| `PERSEO_SMTP_AUTH_USER`   |               | Authentication data, the username                                   |
-| `PERSEO_SMTP_AUTH_PASS`   |               | Authentication data, the user password                              |
-| `PERSEO_NOTICES_PATH`     | `'/notices'`  | Path for incoming notices                                           |
-| `PERSEO_RULES_PATH`       | `'/rules'`    | Path for incoming rules                                             |
+| Environment variable       | Default Value | Description                                                                                                                                                                    |
```

Comment:
Seems like something is messing with this formatting on my end, can't seem to get rid of it

Reply:
Using the "Hide withespace" option is seems to be fine.

![imagen](https://user-images.githubusercontent.com/1534240/143241644-4b5d6024-504a-4363-ae93-e2d368461e21.png)

Reply:
This is useful, thanks!